src/main/scala/beam/router/TransitInitializer.scala

```diff
@@ -44,10 +45,10 @@ class TransitInitializer(
    * be used to decide what type of vehicle to assign
    *
    */
-  def initMap: Map[Id[BeamVehicle], (RouteInfo, ArrayBuffer[BeamLeg])] = {
+  def initMap: ParMap[Id[BeamVehicle], (RouteInfo, Seq[BeamLeg])] = {
```

Comment:
Let's bring back the original `Map` type. When data is ready, you can use `.seq` to convert it to normal Map

Reply:
fixed